tools/orca/src/ipa.rs

```diff
@@ -1,5 +1,5 @@
+use hashbrown::{HashMap, HashSet};
```

Comment:
Isn't this just a cheeky workaround for the `std::collections::HashMap` ban thing? The `hashbrown` crate says it's identical?

> Since Rust 1.36, this is now the HashMap implementation for the Rust standard library.

Reply:
It uses a different hasher though in std lib. hashbrown uses ahash, but std uses siphash which is significantly slower. That's why hashbrown is preferred.